lib/faraday.rb

```diff
@@ -6,7 +6,7 @@
 # Public: This is the main namespace for Faraday.  You can either use it to
 # create Faraday::Connection objects, or access it directly.
 #
-# Examples
+# @example
```

Comment:
Would be great to have 2 separate examples here as well 😃

Reply:
Right, now we have a form, a style of explanation, and we can implement it throughout. 

I can let this PR be about docblocks in this file.